Sources/StreamChatUI/ChatMessageList/ChatMessageListView.swift

```diff
@@ -259,6 +262,28 @@ open class ChatMessageListView: UITableView, Customizable, ComponentsProvider {
         }
     }
 
+    private func getVisibleQuotedMessageUpdates(forUpdatedMessageAt indexPath: IndexPath) -> [IndexPath]? {
```

Comment:
Docs on what this function does would be nice, it's not straightforward from the name